JobConfig/common/mu2eg4.fcl

```diff
@@ -9,7 +9,7 @@ mu2e.physics.producers.g4run.commonBase: {
    debug:  @local::mu2eg4DefaultDebug
    visualization: @local::mu2eg4NoVisualization
 
```

Comment:
What principle are you following to decide if config belongs in Mu2eG4/prolog vs JobConfig/mu2eg4?

Reply:
Configuration required for running G4 test jobs, like g4test_03, should be in the Mu2eG4 directory.   The JobConfig directory may not even exist in a Mu2eG4 code development work area in the future.     On the other hand production-specific configs like dsRegionsCuts  should reside in JobConfig.

Reply:
Separating test configuration from production is a good idea.  In addition, settings specific to a stage should be in that stage's configuration, not common.  I agree NoVisualization is a general production setting.  Flash configuration might eventually fit better in pileup, for now this is fine.